guides/common/modules/proc_configuring-smart-proxy-to-provision-secure-boot-enabled-hosts.adoc

```diff
@@ -0,0 +1,113 @@
+ifeval::["{client-os}" == "Ubuntu"]
+* Ensure that `ar`, `xz`, and `zstd` are installed on your {SmartProxy}.
+endif::[]
+ifeval::["{client-pkg-ext}" == "rpm"]
+* Ensure that `cpio` is installed on your {SmartProxy}.
+endif::[]
+
+.Procedure
+. On your {SmartProxy}, configure the directory to store the shim and GRUB2 binaries required for provisioning Secure Boot enabled hosts:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --foreman-proxy-tftp-bootloader-universe _My_Bootloader_Directory_
+----
++
+Replace _My_Bootloader_Directory_ with the absolute path where you want to store the shim and GRUB2 binaries.
+. Set the path for the shim and GRUB2 binaries for the operating system of your host:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# BOOTLOADER_PATH="_My_Bootloader_Directory_/pxegrub2/_{secureboot-os-name}_/default/x86_64"
+----
++
+If you require specific versions of the shim and GRUB2 binaries for the version of the operating system of your host, replace `default` with the *Major* and *Minor* version of the operating system separated by a dot.
+If no *Minor* version is set, replace `default` with the *Major* version.
++
+{Team} recommends to not use version-specific shim and GRUB2 binaries unless it is really necessary.
+. Create the directory to store the shim and GRUB2 binaries for the operating system of your host:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# install -o foreman-proxy -g foreman-proxy -d $BOOTLOADER_PATH
+----
+. Download the shim and GRUB2 packages for the operating system of your host:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# wget -O /tmp/{grub_efi_downloaded_package_name} _https://{server-example-com}/{grub_efi_downloaded_package_name}_
+# wget -O /tmp/{shim_efi_downloaded_package_name} _https://{server-example-com}/{shim_efi_downloaded_package_name}_
+----
++
+You can download the `{grub_efi_package_name}` package from {grub_efi_download_url}.
+You can download the `{shim_efi_package_name}` package from {shim_efi_download_url}.
+. Extract the shim and GRUB2 binaries:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# {extract_grub}
+# {extract_shim}
+----
+. Make the shim and GRUB2 binaries available for host provisioning:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# cp {grub_efi_tmp_binary_path} $BOOTLOADER_PATH/grubx64.efi
+# cp {shim_efi_tmp_binary_path} $BOOTLOADER_PATH/shimx64.efi
+# ln -sr $BOOTLOADER_PATH/grubx64.efi $BOOTLOADER_PATH/boot.efi
+# ln -sr $BOOTLOADER_PATH/shimx64.efi $BOOTLOADER_PATH/boot-sb.efi
+# chmod 644 $BOOTLOADER_PATH/grubx64.efi
+# chmod 644 $BOOTLOADER_PATH/shimx64.efi
+----
+ifeval::["{client-pkg-ext}" == "deb"]
+. Link the `grub.cfg` file from the TFTP servers `grub2` folder to the legacy `grub` folder:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# ln -rs /var/lib/tftpboot/grub2/grub.cfg /var/lib/tftpboot/grub/grub.cfg
```

Comment:
TIL: `-r` to make a link relative.

I always prefer long options to make it self-documenting

```suggestion
# ln --relative --symbolic /var/lib/tftpboot/grub2/grub.cfg /var/lib/tftpboot/grub/grub.cfg
```